template.html

```diff
@@ -1,50 +1,83 @@
-<!doctype html>
-
-<html>
-
+<!DOCTYPE html>
+<html lang="en">
 <head>
-
-	<meta charset="UTF-8">
-
-	<title>${head_title}</title>
-
-	<meta name="description" content="${description}">
-
-	<link rel="canonical" href="https://book.cuberite.org">
-
-	<link rel="stylesheet" href="css/main.css" media="screen">
-	<link rel="stylesheet" href="css/print.css" media="print">
-
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Manual - Cuberite</title>
+	<meta name="description" content="A custom Minecraft compatible game server written in C++. It's a fast, lightweight, and awesome replacement for Vanilla.">
+	<link rel="canonical" href="https://book.cuberite.org/">
+	<link rel="stylesheet" href="https://cuberite.org/css/style.css">
```

Comment:
We should really import the files into this repository, or break the CSS out into a seperate repo and add a submodule. The book should be self-contained so you can read it without a network connection to the cuberite site.